chainerx_cc/chainerx/backward.cc

```diff
@@ -590,39 +597,67 @@ std::vector<nonstd::optional<Array>> Grad(
         const std::vector<ConstArrayRef>& outputs,
         const std::vector<ConstArrayRef>& inputs,
         const nonstd::optional<BackpropId>& backprop_id,
-        DoubleBackpropOption double_backprop) {
+        DoubleBackpropOption double_backprop,
+        bool set_grad,
+        bool retain_grad,
+        const std::vector<ConstArrayRef>& grad_inputs,
```

Comment:
I wonder if we can omit `grad_inputs` argument: https://github.com/chainer/chainer/issues/5978#issuecomment-453090989

Reply:
I agree with removing it, I just added to make chainer and chainerx grad interface to be consistent.

Reply:
Please remove it for now. We may add it later if it's really needed.

Reply:
Ok!